infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/util/LinuxInitScriptGenerator.java

```diff
@@ -70,7 +70,12 @@
     }
 
     public String generateNodeDownloadCommand(String rmHostname) {
-        return "wget -nv " + rmHostname + ":8080/rest/node.jar";
+        if (System.getProperty("os.name").contains("Windows")) {
+            return "powershell -command \"& { (New-Object Net.WebClient).DownloadFile('" + rmHostname +
```

Comment:
I have a doubt: as this function (`generateNodeDownloadCommand`) is executed on the **proactive server**, while the generated command will be executed on the **node**, I think this download command should depend on the os of the node instead of the server.

Reply:
Can you validate please? I will check how to do for this scenario in another card